contributing.md

```diff
@@ -0,0 +1,16 @@
+# Contributing to `ihmeuw-demographics` repositories
+
+The demographics team for the [Global Burden of Disease Study](http://www.healthdata.org/gbd) at the [Institute for Health Metrics and Evaluation](http://www.healthdata.org/) is actively working on moving more code development to publicly available github repositories.
+
+We encourage contributions of bug reports, feature requests, improved documentation, and software development. As this open source project is still in its infancy, documentation and testing has mainly been created and reviewed by GBD demographics team members, so we appreciate feedback in making this work more accessible to collaborators outside of IHME.
```

Comment:
Be more explicit that this can be (a) submit a GitHub issue, or, (b) make a change yourself and submit a pull request.

We should also be clear with respect to pull requests, the expectations and limitations set on contributors who are not IHME demographics team members.

Reply:
Can you check the change I just pushed?

Reply:
Looks good